### ISSUE TITLE
regression: files with unsupported characters on Windows silently ignored (2.1) #5076

### ISSUE THREAD

**Issue by moscicki** [owncloud/client#5963]
There appears to be a regression in 2.2.4 and 2.3.1 #5076 

Could you please confirm?

**Comment by guruz** [comment]
@moscicki Current version is 2.3.3rc1 :-) Re-try that first..

**Comment by yolanda93** [comment]
We have tried with the current version (2.3.3rc1) and the problem remains

**Comment by michaelstingl** [comment]
@moscicki @yolanda93 Could you send the exact "steps to reproduce" to better cover it in qa?

**Comment by SamuAlfageme** [comment]
@michaelstingl no need, I could reproduce by simply including a colon on any filename via webUI and forcing/waiting for sync. 

@ogoffart @guruz wonder if the new encoding/decoding logic (https://github.com/owncloud/client/pull/5875) could have some impact here. Probably that is making the error to go unnoticed.

**Comment by guruz** [comment]
I'm creating a file called ```:```  on the web interface with oC 9.1.6.2 and Windows 10.x

I can then clearly see it in ```Not Synced``` tab as ```File names containing the character ':' are not supported on this file system```.

@moscicki @yolanda93 Can you give me more detailed reproduction steps?
Are you sure you waited for the next sync to happen and then checked the ```Not Synced``` tab?
It would be good to run the client with log file and paste a link to log file here.

Now the big question is of course how was @SamuAlfageme able to reproduce!?
Is it because you all have different oC versions?
But I thought for downloading those checks are client side..

**Comment by SamuAlfageme** [comment]
@guruz 

> But I thought for downloading those checks are client side..

Indeed; you're right, I'm seeing the entries appear on "`Not Synced`" after a few seconds, I think the other day I assumed **not seeing the file-count** in the tab meant there was nothing there (I'm used to it on 2.4 by now):

| 2.3.X | 2.4.0 |
|---|---|
| ![screenshot 2017-08-24 12 35 44](https://user-images.githubusercontent.com/2644445/29663523-d1acbaca-88cb-11e7-874f-939fba96cae8.png) | ![screenshot 2017-08-24 12 59 54](https://user-images.githubusercontent.com/2644445/29663632-5a1070dc-88cc-11e7-8c08-c48769a46be3.png) |

@moscicki @yolanda93 do you guys have any more insights on how to reproduce it?

**Comment by guruz** [comment]
@moscicki @yolanda93  ping ^^

**Comment by yolanda93** [comment]
The same as @guruz, sorry. Now, I double checked waiting a few seconds and it appears in Not Synced tab.

**Comment by guruz** [comment]
@yolanda93 great!

@ogoffart @mrow4a Just a thought we could have those kind of error jobs (generated by a sync job, not in discovery) be ordered very early so that they are visible earlier in the UI?
I don't know how this is influenced by renames/moves

**Comment by mrow4a** [comment]
Hmm, I think we can do that, any priority for that?
